src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
```

Comment:
The DB_URL is incomplete and it's missing the database server address, port, and database name. Make sure to provide the full JDBC URL in the following format: `jdbc:mysql://<server>:<port>/<database_name>`.